uwg_schema/ref_bld_template.py

```diff
@@ -268,46 +274,42 @@ class BEMDef(NoExtraBaseModel):
         description='Element object for building roof.'
     )
 
-    frac: float = Field(
-        0,
-        ge=0,
-        le=1,
-        description='Fraction of the urban floor space of this typology.'
-    )
-
 
 class SchDef(NoExtraBaseModel):
     """Schedule definition class."""
 
     type: constr(regex='^SchDef$') = 'SchDef'
 
-    bldtype: int = Field(
+    bldtype: str = Field(
         ...,
-        ge=0,
-        description='Number between 0 and 15 corresponding to the following '
-        'building types: FullServiceRestaurant (0), Hospital (1), LargeHotel (2), '
-        'LargeOffice (3), MediumOffice (4), MidRiseApartment (5), OutPatient (6), '
-        'PrimarySchool (7), QuickServiceRestaurant (8), SecondarySchool (9), '
-        'SmallHotel (10), SmallOffice (11), StandaloneRetail (12), StripMall (13), '
-        'SuperMarket (14), Warehouse (15). Additional building types can be defined '
-        'with a number greater then 15. This value is used to reference the fraction '
-        'of urban area the SchDef object defines in the UWG bld matrix.'
+        description='Text referring to a building type. To reference (or overwrite) '
+        'a DOE reference building, text must be one of the following: '
```

Comment:
It doesn't have to be one of these, right?  It just has to be unique and correspond to a building type that's under the `'bld'` key, right?

Reply:
Yeap, that's what that text is saying: you must use the names in that list ONLY if you want to reference or overwrite a DOE reference.  I can add a line saying unique ids are allowed to make it clearer.